include/onnxruntime/core/providers/cpu/cpu_provider_factory.h

```diff
@@ -10,6 +10,7 @@ extern "C" {
 /**
  * \param use_arena zero: false. non-zero: true.
  */
+ORT_EXPORT
```

Comment:
So far CPU and CoreML are only 2 EPs can be used on mac, also added it for NNAPI since we can build NNAPI EP for model conversion.

Not sure if we want to add ORT_EXPORT to all EPs